gradle/libs.versions.toml

```diff
@@ -44,6 +46,9 @@ androidx-compose-ui-tooling = { module = "androidx.compose.ui:ui-tooling" }
 androidx-compose-ui-tooling-preview = { module = "androidx.compose.ui:ui-tooling-preview" }
 androidx-compose-ui-util = { module = "androidx.compose.ui:ui-util" }
 
+# https://github.com/google/TestParameterInjector
+google-testParameterInjector = { module = "com.google.testparameterinjector:test-parameter-injector", version.ref = "testParameterInjector" }
```

Comment:
IMO we should not prefix everything as it makes it harder to find what you look for.
I think we should keep `testParameterInjector` and `paparazzi`

Reply:
Done in a792de44